recipes/nlp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "nlp" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a2b0f469a29d0dd6235a1ab8c218bf745c904ba9fdf9cd150c52ba28de345b81
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - dill
+    - filelock
+    - numpy
+    - pyarrow ==0.17.1
```

Comment:
Does this pin need to be so tight?

Reply:
Yeah. It has to be this or `==0.17`. Not sure why though. Only `>=0.16` is needed by nlp.